containerz/containerz.proto

```diff
@@ -299,3 +327,69 @@ message LogRequest {
 message LogResponse {
   string msg = 1;
 }
+
+message CreateVolumeRequest {
+  // name of the volume. If present, this system must create the volume with
+  // this name otherwise it must allocate a name.
+  string name = 1;
+
+  // The volume driver. If not specified, this defaults to the the standard
```

Comment:
Can this be an enumerated value? It seems like here we're very flexible, and there's maybe not a clear definition that will work across different implementations. It's also not clear how to express the minimum that `Containerz` is looking for -- e.g., is anything other than `MOUNT` actually required?